content/kubermatic/main/tutorials-howtos/networking/apiserver-policies/_index.en.md

```diff
@@ -39,3 +40,16 @@ spec:
 ```
 
 Please note that this procedure does not affect already running user clusters, for those the API Server Network Policies need to be disabled individually as described in the previous section.
+
+## API Server Allowed Source IP ranges
+Since KKP v2.22, it is possible to restrict the access to the user cluster API server based on the source IP ranges. To restrict the server access from the internet or to have a limited access, `front-loadbalancer` k8s service can be used to allow only the configured IP source ranges. It can be confiured as `.spec.loadBalancerSourceRanges`. This works only when expose strategy is set to `LoadBalancer`.  
```

Comment:
I would maybe avoid these sentences, as it describes KKP internals and is not so important for the users who just want to use the feature - and may even mislead them:

> To restrict the server access from the internet or to have a limited access, `front-loadbalancer` k8s service can be used to allow only the configured IP source ranges. It can be confiured as `.spec.loadBalancerSourceRanges`